pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "epcsunspecdemo"
+version = "0.1.0.post46.dev0+2a57176"
+description = ""
+authors = ["Alex Anker <alex.anker@epcpower.com>"]
+
+[tool.poetry-dynamic-versioning]
+enable = true
+
+
+[tool.poetry.dependencies]
+python = "3.7.9"
+attrs = "18.2.0"
+click = "7"
+gitignoreio = "0.1.5"
+toolz = "0.9.0"
+tqdm = "4.32.1"
+pysunspec = "2.1.0"
+setuptools = "53.0.0"
+toml = "^0.10.2"
+pyinstaller = "4.2"
```

Comment:
Wouldn't `pyinstaller` be under `tool.poetry.dev-dependencies`? Maybe I am misunderstanding the meaning of "dev-dependencies"?